lib/DataIntegrityProof.js

```diff
@@ -262,7 +275,10 @@ export class DataIntegrityProof extends LinkedDataProof {
    *
    * @returns {Promise<{object}>} Resolves with the verification result.
    */
-  async verifyProof({proof, proofSet, document, documentLoader}) {
+  async verifyProof({
+    proof, proofSet,
+    document, documentLoader,
+  }) {
```

Comment:
Can you revert this unnecessary change?

Reply:
reverted here: https://github.com/digitalbazaar/data-integrity/pull/34/commits/0f66109ccc5da09d7c79ad5170f97c119525ef8e